scripts/upgrade-runtime.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -xEe
+
+if [[ -z "$1" || -z $2 || -z $3 ]]; then
+  echo "usage: $0 \"'sudo secret'\" \$provider \$wasm [\$when]"
+  # fx: ./upgrade-runtime.sh "'sudo secret'" wss://dev.net.t3rn.io ../specs/t0rn.wasm
+  exit 1
+fi
+
+sudo_secret="$1"
+provider=$2
+wasm=$3
+when=$4
+
+if [[ -n $when ]]; then
+  echo "scheduled runtime upgrades not supported yet" >&2
+  exit 1
+fi
+
+read -n 1 -p "runtime tested and incremented spec version? (y/n) " answer
+
+echo
+
+if [[ "${answer,,}" != "y" ]]; then exit 1; fi
+
+npx --yes @polkadot/api-cli@beta \
+  --ws $provider \
+  --sudoUncheckedWeight \
+  --seed "\"$sudo_secret\"" \
+  tx.system.setCode \
```

Comment:
tx.scheduler.schedule?

Reply:
scheduling is pending (see issue above)
this is the early bike shed version